.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -27,10 +27,11 @@ KFP SDK version: <!-- Please attach the output of this shell command: $pip list
 
 /kind bug
```

Comment:
You can use `labels:` to configure this now

Reply:
Sounds good! Moved to use labels.